src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -770,11 +771,11 @@ export class PointerToolScale extends PointerTool {
 
   activate() {
     this.setCursor();
-    this.sceneController.experimentalFeatures.scalingEditBehavior = true;
+    this.scalingEditBehavior = true;
```

Comment:
This should not be a dynamic property: in `PointerTool` `this.scalingEditBehavior` should always be false, and in `PointerToolScale` it should always be true. Since we should not write to this property, perhaps it should be a getter, like how it's done in the `curveType` getter in the pen tools.

Reply:
Ah, yes, you're right. That's much better. Done.